modules/fedimint-lnv2-server/src/lib.rs

```diff
@@ -544,6 +557,37 @@ impl ServerModule for Lightning {
                     Ok(module.outgoing_contract_expiration(db, contract_id).await)
                 }
             },
+            api_endpoint! {
+                ADD_GATEWAY_ENDPOINT,
```

Comment:
Should we generally name more stuff here with "vetted" word in it? endpoints, some constants, db fields. I guess that's the meaning and it greatly clarifies the context here.

Reply:
I am open to it.